YouVsVirus/Assets/Scripts/UI/Levelcollectmasks/masktext.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+namespace Components
+{
+
+public class masktext : MonoBehaviour
```

Comment:
Should begin with a capital letter?

Reply:
Wouldn't it be much easier if the FaceMask.cs would get the mask text as game object and could update it as soon as the player has the mask?

Reply:
> I think most scripts are now in the same folder as the scenes, you have separated it. What do you prefer?

mhmm, i am a bit confused here.
We have a folder Scripts, which we should use for scripts.
But there are sometimes very level specific scripts und sometimes scripts that were introduced in a level but during develop get the status of scripts that are needed everywhere.

I would suggest that we properly discuss this again after the release and clean up our folder structure.